html/elements/iframe.json

```diff
@@ -1140,6 +1140,65 @@
               "deprecated": false
             }
           }
+        },
+        "mozbrowser": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Using_the_Browser_API",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": [
+                  "See <a href='https://bugzil.la/1318532'>bug 1318532</a>",
+                  "Available only to <a href='https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions'>WebExtensions</a>."
```

Comment:
This is not strictly true; it has been implemented in WebExtensions, but I believe it is available in privileged code/chrome code in general, and there were uses of it cited for DevTools too?

Reply:
See https://github.com/mdn/browser-compat-data/pull/2575, which also exposed a chrome code only property to WebExtensions, but the chrome code access is not mentioned, as for most MDN readers, it’s unnecessary noise.

needinfo?(@Elchi3): what do you think about this?

Reply:
I think MDN does not document chrome code, nor does BCD. 
Instead, MDN documents open web features and cross-browser webextensions.

Reply:
Good point. In that case, let's not worry about mentioning it. r+